web/package.json

```diff
@@ -26,7 +26,7 @@
     "node-fetch": "^2.6.7",
     "object-hash": "^2.2.0",
     "pino": "^6.13.0",
-    "pino-pretty": "^7.3.0",
+    "pino-pretty": "5.1.3",
```

Comment:
this was bumped by dependabot very recently and it caused things to break in dev mode, so i reverted it for now